### ISSUE TITLE
[Bug]: Expected kernel size to be equal to input size - 2. Got: 3

### ISSUE THREAD

**Issue by jikechao** [openvinotoolkit/openvino#20907]
### OpenVINO Version

openvino-nightly 2023.2.0.dev20231101

### Operating System

Ubuntu 18.04 (LTS)

### Device used for inference

BATCH

### Framework

PyTorch

### Model used

Given in the following script

### Issue description

OpenVINO Convert the PyTorch model with `avg_pool3d` failed when the input dimension was not equal to 5.
For the avg_pool3d in PyTorch, it supports input data with any input dimensions. 
Will OpenVINO support 4-dims inputs (e.g, NCHW) for `pool3d` in the future?

### Step-by-step reproduction

```
import torch
import openvino as ov
from torch.nn import Module


class avg_pool3d(Module):
    def forward(self, *args):
        return torch.nn.functional.avg_pool3d(args[0], kernel_size=2)


input_data = torch.randn([1, 5, 6, 7], dtype=torch.float32)
torch_model = avg_pool3d().float().eval()
trace = torch.jit.trace(torch_model, input_data)
trace = torch.jit.freeze(trace)
input_shapes = input_data.shape
print(input_shapes)
torch_outputs = torch_model(input_data).cpu().numpy()
print(torch_outputs.shape)

ov_model = ov.convert_model(trace, example_input=input_data)
ir_path = f"temp_OVIR.xml"
ov.save_model(ov_model, ir_path, compress_to_fp16=False)
core = ov.Core()
model = core.read_model(ir_path)

compiled_model = core.compile_model(model=model, device_name="CPU")
output_key = compiled_model.output(0)
result = compiled_model(input_data)[output_key]

```

### Relevant log output

```shell
Traceback (most recent call last):
  File "test.py", line 20, in <module>
    ov_model = ov.convert_model(trace, example_input=input_data)
  File "C:\software\conda\envs\torch\lib\site-packages\openvino\tools\ovc\convert.py", line 101, in convert_model
    ov_model, _ = _convert(cli_parser, params, True)
  File "C:\software\conda\envs\torch\lib\site-packages\openvino\tools\ovc\convert_impl.py", line 524, in _convert
    raise e
  File "C:\software\conda\envs\torch\lib\site-packages\openvino\tools\ovc\convert_impl.py", line 476, in _convert
    ov_model = driver(argv, {"conversion_parameters": non_default_params})
  File "C:\software\conda\envs\torch\lib\site-packages\openvino\tools\ovc\convert_impl.py", line 226, in driver
    ov_model = moc_emit_ir(prepare_ir(argv), argv)
  File "C:\software\conda\envs\torch\lib\site-packages\openvino\tools\ovc\convert_impl.py", line 172, in prepare_ir
    ov_model = moc_pipeline(argv, moc_front_end)
  File "C:\software\conda\envs\torch\lib\site-packages\openvino\tools\ovc\moc_frontend\pipeline.py", line 247, in moc_pipeline
    ov_model = moc_front_end.convert(input_model)
  File "C:\software\conda\envs\torch\lib\site-packages\openvino\frontend\frontend.py", line 18, in convert
    converted_model = super().convert(model)
openvino._pyopenvino.OpConversionFailure: Check 'is_conversion_successful' failed at src\frontends\pytorch\src\frontend.cpp:138:
FrontEnd API failed with OpConversionFailure:
Model wasn't fully converted. Failed operations detailed log:
-- aten::avg_pool3d with a message:
Exception happened during conversion of operation aten::avg_pool3d with schema aten::avg_pool3d(Tensor self, int[3] kernel_size, int[3] stride=[], int[3] padding=0, bool ceil_mode=False, bool count_include_pad=True, int? divisor_override=None) -> Tensor
Check 'data_rank.is_dynamic() || num_spatial == (data_shape.size() - spatial_dim_offset)' failed at src\core\shape_inference\include\pooling_shape_inference_util.hpp:56:
While validating node 'opset1::AvgPool AvgPool_24 (aten::avg_pool3d/Pad[0]:f32[?,?,?,?]) -> (dynamic[...])' with friendly_name 'AvgPool_24':
Expected kernel size to be equal to input size - 2. Got: 3

-- prim::Constant with a message:
None constant cannot be converted to OpenVINO opset and should be removed by consuming operation.
Summary:
-- Conversion is failed for: aten::avg_pool3d, prim::Constant
```

### Issue submission checklist

- [x] I'm reporting an issue. It's not a question.
- [X] I checked the problem with the documentation, FAQ, open issues, Stack Overflow, etc., and have not found a solution.
- [X] There is reproducer code and related data files such as images, videos, models, etc.

**Comment by mvafin** [comment]
@jikechao Thank you for the issue. This is a known problem when 3d pooling operation gets 4-dimensional input and we have plans to fix it. However we haven't seen yet any model that use this pattern, so the priority of the fix is low at the moment.

**Comment by mvafin** [comment]
Will be fixed by #20927
